src/lib/const.ts

```diff
@@ -55,6 +56,35 @@ export const defaultNumberProperties: Record<NumberProperty, number> = {
   minPricePerWorkerEpoch: PRICE_PER_EPOCH_DEFAULT,
 };
 
+export const PER_WORKER_MEMORY_LIMIT_STR = "2GB";
+export const PER_WORKER_MEMORY_LIMIT = xbytes.parseSize(
+  PER_WORKER_MEMORY_LIMIT_STR,
+);
+
+const byteUnits = [
+  "kB",
+  "KB",
+  "kiB",
+  "KiB",
+  "KIB",
+  "mB",
+  "MB",
+  "miB",
+  "MiB",
+  "MIB",
+  "gB",
+  "GB",
+  "giB",
+  "GiB",
+  "GIB",
+];
+
+export const BYTES_PATTERN = `^\\d+(\\.\\d+)?(\\s?)(${byteUnits.join("|")})$`;
+export const BYTES_FORMAT = `[number][whitespace?][B] where ? is an optional field and B is one of the following: ${byteUnits.join(
+  ", ",
+)}`;
+export const MAX_HEAP_SIZE_DESCRIPTION = `Max size of the heap that a module can allocate in format: ${BYTES_FORMAT}`;
```

Comment:
It's very hard to say in js when something deserves a separate file. Probably it's good to have a lot of small files in a sense that it's possible to contain imports sometimes and not Node.js will not have to parse a bunch of stuff it will not use. But alternatively there can become too much of the files. and pretty messy dependency tree. So it's really hard to tell. For cli I have const file where all the consts live. Maybe it can be split somehow, and it will increase number of imports as well. So I am not really sure about this issue, more of a matter of taste